source/Application/Component/UserComponent.php

```diff
@@ -260,7 +260,7 @@ protected function _afterLogin($oUser)
      * Executes oxcmp_user::login() method. After loggin user will not be
      * redirected to user or payment screens.
```

Comment:
I think it's cleaner to ignore these errors in the docblock, like:
```suggestion
     * redirected to user or payment screens.
     *
     * @phpcs:disable PSR1.Methods.CamelCapsMethodName.NotCamelCaps
```
You can find the violated rule by adding the `-s` parameter to `phpcs`.

The benefit is, that it does not break static code analysis.

Reply:
Hey @alfredbez,

thanks. Sadly phpcs is not working with PHPDoc annotations. So this change would disable this rule from that line onward till the end of the file, which is not what we want. But I added the rule after the `phpcs:ignore` comment, see commit https://github.com/OXID-eSales/oxideshop_ce/pull/769/commits/1b9e4a2eb1f0b4711e09c8655990ebbfcdee1cad

Could you further explain what you mean by

> does not break static code analysis

As I did not see any difference in the report from PHPStan with or without that `phpcs:ignore` comment on the `function` line.

/Flo

Reply:
You're right, I just saw this in https://github.com/squizlabs/PHP_CodeSniffer/issues/2367.

The comment shouldn't be a problem for static code analyzers either, just tested it here: https://psalm.dev/r/1efb47d890

It is only a problem if you make the comment on a seperate line between the docblock and the function like here: 
https://psalm.dev/r/1c5ba583c6